bin/node/src/commands/store.rs

```diff
@@ -95,31 +100,53 @@ impl StoreCommand {
             .context("failed while serving store component")
     }
 
-    fn bootstrap(data_directory: &Path, accounts_directory: &Path) -> anyhow::Result<()> {
-        // Generate the genesis accounts.
-        let account_file =
-            Self::generate_genesis_account().context("failed to create genesis account")?;
-
-        // Write account data to disk (including secrets).
-        //
-        // Without this the accounts would be inaccessible by the user.
-        // This is not used directly by the node, but rather by the owner / operator of the node.
-        let filepath = accounts_directory.join(DEFAULT_ACCOUNT_PATH);
-        File::create_new(&filepath)
-            .and_then(|mut file| file.write_all(&account_file.to_bytes()))
-            .with_context(|| {
-                format!("failed to write data for genesis account to file {}", filepath.display())
-            })?;
-
-        // Bootstrap the store database.
-        let version = 1;
-        let timestamp = SystemTime::now()
-            .duration_since(UNIX_EPOCH)
-            .expect("current timestamp should be greater than unix epoch")
-            .as_secs()
-            .try_into()
-            .expect("timestamp should fit into u32");
-        let genesis_state = GenesisState::new(vec![account_file.account], version, timestamp);
+    fn bootstrap(
+        data_directory: &Path,
+        accounts_directory: &Path,
+        maybe_genesis_config_path: Option<&PathBuf>,
```

Comment:
Nit: I would just call this `genesis_config`, feels bit like hungarian notation otherwise.

Reply:
No strong opinion, helps me understand code faster, even before `rust-analyzer` is ready.

Reply:
Changed to `config_path`